eventstore/store.go

```diff
@@ -93,10 +93,23 @@ func (s *EventStore) Init() error {
 			return err
 		}
 	}
-	if err := s.refresh(tx); err != nil {
+	if err := tx.Commit(); err != nil {
 		return err
 	}
-	if err := tx.Commit(); err != nil {
+	// create the ephemeral configuration objects (pricing/plans/etc)
```

Comment:
Generally we avoid using comments as prolifically as this - the code should speak for itself. I think most of these new comments don't add much value, and may drift out-of-sync with what the code does (at which point they'd be actively harmful).